app/dashboard/static/js/app/view-tests-suite-id.2017.7.2.js

```diff
@@ -0,0 +1,557 @@
+/*!
+ * kernelci dashboard.
+ * 
+ * Copyright (C) 2014, 2015, 2016, 2017  Linaro Ltd.
```

Comment:
You should add Baylibre here, or use the short version of the license header (although it might not be compliant, and we should have a COPYING file somewhere).

Reply:
Added Baylibre and author